go-for-eat/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import { View, Text, Image, Avatar, TouchableOpacity } from 'react-native';
+import { Header } from 'react-native-elements';
+import { connect } from 'react-redux';
+
+import { logoutUser, navigate, navigateBack } from '../../actions';
+import { logoutStorage } from '../../localStorage'
+
+import navNew from '../../assets/icons/nav_new.png';
+import navBack from '../../assets/icons/nav_back.png';
+import navProfile from '../../assets/icons/nav_profile.png';
+import navLogout from '../../assets/icons/nav_logout.png';
+import navClose from '../../assets/icons/nav_close.png';
+import logo from '../../assets/logo/logo2x.png';
+import style from './styles.js';
+import PropTypes from 'prop-types';
+
+class NavBar extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {screen: this.props.screen}
```

Comment:
why are you mapping the state to the props if you already have the props and they behave basically the same as the state?